src/libecalc/core/consumers/pump/component.py

```diff
@@ -43,15 +44,17 @@ def get_max_rate(self, operational_settings: PumpOperationalSettings) -> List[fl
             operational_settings_this_timestep = operational_settings.get_subset_for_timestep(timestep)
             results.extend(
                 pump.get_max_standard_rate(
-                    suction_pressures=np.asarray(operational_settings_this_timestep.inlet_pressure.values),
-                    discharge_pressures=np.asarray(operational_settings_this_timestep.outlet_pressure.values),
-                    fluid_density=np.asarray(operational_settings_this_timestep.fluid_density.values),
+                    suction_pressures=np.asarray(operational_settings_this_timestep.inlet_streams[0].pressure.values),
```

Comment:
first or list of lists? Ref ole petters comment

Reply:
multistream isn't handled yet, first because that is the requested stream and where the target pressure is set.